plugins/katello/nightly/user_guide/activation_keys/index.md

```diff
@@ -86,24 +90,19 @@ Finally, when 'Auto-Attach' is disabled, all subscriptions on the activation key
 
 ![Adding subscriptions to an Activation Key](/plugins/katello/{{ page.version }}/user_guide/activation_keys/activation_key_add_subscriptions.png)
 
-
-
 ## Change Repository Enablement for an Activation Key
 
 To change repository enablement settings using a key:
 
 - navigate to: Content > Activation Keys
 - select the desired key from the list
-- click **Product Content**
-- click the edit icon for the repository content set that you would like to modify
-- select the desired value (e.g. 'Override to Yes', 'Override to No', 'Defaults to Yes', 'Defaults to No')
-- click **Save**
+- click **Repository Sets**
+- select the repository that you would like to modify
+- select the desired value under the **Select Action** drop down(e.g. 'Override to Enabled', 'Override to Disable', 'Reset to Default')
```

Comment:
Avoid "desire" in technical writing: s/select the desired value under the **Select Action** drop down(e.g. 'Override to Enabled', 'Override to Disable', 'Reset to Default')/select the value you want to change from the **Select Action** menu ('Override to Enabled', 'Override to Disable', or 'Reset to Default')/  {no need to use e.g. when you are giving a complete list}